doc/maintenance/announce_outage.rst

```diff
@@ -0,0 +1,78 @@
+.. _announcing_fedora_copr_outage:
+
+Fedora Copr Outages
+===================
```

Comment:
The title is "Fedora Copr Outages" but in the Maintenance Documentation page it is listed as "Outage announcements". Can you please change one of those to have consistency?

Reply:
Right, changed both.